owl/EASE-OBJ.owl

```diff
@@ -2564,7 +2596,9 @@
     <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Sharpness -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Sharpness">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#GeometricalQuality"/>
+        <rdfs:comment>The quality of having a thin edge or point that can cut something or make a hole in something. It is worth to note here that the social aspect of sharpness such as the quality of being clear, intelligent etc is not considered as sharpness according to this definition.</rdfs:comment>
+        <rdfs:seeAlso>https://dictionary.cambridge.org/dictionary/english/sharpness</rdfs:seeAlso>
```

Comment:
I think we have discussed that we do not want to have links to dictionaries, or?